cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/ConfigReview/ConfigReview.scss

```diff
@@ -1,45 +1,45 @@
 .config-section {
-  font-size: var(--cds-global-typography-font-size-6);
-  font-weight: bold;
-  vertical-align: center;
+    font-size: var(--cds-global-typography-font-size-6);
```

Comment:
As a team, we should probably sync on indentation defaults so we don't go back and forth on spacing issues

Reply:
Yes. I will use the former one.

Reply:
Found that in other *.scss files, the indention is 4 blanks. 
So do we want to make this as 2 blanks particularly?

Reply:
No, if the other *.scss files have four blanks, then it's probably my own IDE configuration that needs adjusting!